src/notebook/notebookKernel.ts

```diff
@@ -21,6 +21,9 @@ const notifyTypeRunCell = new NotificationType<{ current_request_id: number, cod
 const notifyTypeRunCellSucceeded = new NotificationType<{ request_id: number }>('runcellsucceeded')
 const notifyTypeRunCellFailed = new NotificationType<{ request_id: number, output: {ename: string, evalue: string, traceback: string}}>('runcellfailed')
 
+function getDisplayPathName(pathValue: string): string {
+    return pathValue.startsWith(homedir()) ? `~${path.relative(homedir(), pathValue)}` : pathValue
```

Comment:
Display relative paths to home directory (else the paths are just unnecessarily long). & users in general know that `~/sdfwef` means its relative to home